### PR TITLE
Bicycle is now a bicycle! (and gives it a bicycle bell too)

### PR DESCRIPTION
**Pull request by Henri215**
## What Does This PR Do

Adds a new sprite to the bicycle.
Bicycle now has a bicycle bell that you can play while moving! (1 second cooldown)

Bicycle is now a vehicle instead of a wheelchair.

## Why It's Good For The Game

image
Bicycle is not a motorcycle.

A player that did farm 10k tickets deserves a bicycle with a bell.
## Images of changes

https://github.com/ParadiseSS13/Paradise/assets/77684085/3ed24042-9e37-45b0-9e08-2572b0af029f

image

## Testing

- See video above
## Changelog
:cl: alexioshq, Henri215
tweak: The bicycle now has a bicycle bell!
imageadd: Added a bicycle sprite to the bicycle
/:cl:

### CODE REVIEW

**Comment by Henri215** [comment]
https://github.com/ParadiseSS13/Paradise/assets/77684085/18366cd6-c5a8-430c-b303-a7706eb76875

This is what our current "bike" looks like.

**Review by DGamerL — COMMENTED** [review]
At a glance, the code looks good, but in the video it looks like the bicycle has no front facing sprite?

**Comment by DGamerL** [comment]
> > At a glance, the code looks good, but in the video it looks like the bicycle has no front facing sprite?
> 
> image image
> 
> There, two images for you. (and the TG sprite has a front facing sprite, it is just really small)

What's draft do again?

**Comment by Burzah** [comment]
> > > At a glance, the code looks good, but in the video it looks like the bicycle has no front facing sprite?
> > 
> > 
> > image image
> > There, two images for you. (and the TG sprite has a front facing sprite, it is just really small)
> 
> What's draft do again?

I think it is meant to say drift. He is going to make the bikes drift around corners!

**Comment by Henri215** [comment]
https://github.com/ParadiseSS13/Paradise/assets/77684085/9e9da389-8704-4635-a186-1c74d37b961e

New version of the coder sprite.

File: code/modules/vehicle/bicycle.dm [hunk 1]
```diff
@@ -0,0 +1,62 @@
+/obj/vehicle/bike
+	name = "bicycle"
+	desc = "Two wheels of FURY!"
+	icon = 'icons/obj/vehicles.dmi'
+	icon_state = "bicycle"
+	vehicle_move_delay = 1
```
Comment (DGamerL) [inline code/modules/vehicle/bicycle.dm:6]: Ok but what if badass people were like slightly faster on a bike

File: code/modules/vehicle/bicycle.dm [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+		else if(E.status & ORGAN_BROKEN)
+			vehicle_move_delay += 1.5
+
+/obj/vehicle/bike/post_buckle_mob(mob/living/M)
+	. = ..()
+	var/datum/action/bicycle_bell/bell_action = new(src)
+	if(has_buckled_mobs())
+		bell_action.Grant(M)
+	else
+		bell_action.Remove(M)
```
Comment (DGamerL) [inline code/modules/vehicle/bicycle.dm:37]: When would we have this? Wouldn't this be handled by `post_unbuckle_mob`?

File: code/modules/vehicle/bicycle.dm (hunk 1 above)
Comment (Henri215) [reply code/modules/vehicle/bicycle.dm:6]: Hmm, maybe in another PR since you could apply that logic to other vehicles too.

Comment (DGamerL) [reply code/modules/vehicle/bicycle.dm:6]: Fair enough! Lewc Vehicle rework 2023!

File: code/modules/vehicle/bicycle.dm (hunk 2 above)
Comment (Henri215) [reply code/modules/vehicle/bicycle.dm:37]: Removed, it was truly not necessary from investigating it now.

**Review by DGamerL — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Review by DGamerL — APPROVED** [review]
Bike real 2023

**Review by S34NW — APPROVED** [review]

**Pull request merged**